resources/js/Pages/Collections/Components/CollectionsFilter/CollectionsNetworksFilter.tsx

```diff
@@ -0,0 +1,83 @@
+import cn from "classnames";
+import React from "react";
+import { useTranslation } from "react-i18next";
+import { Checkbox } from "@/Components/Form/Checkbox";
+import { NetworkIcon } from "@/Components/Networks/NetworkIcon";
+
+interface Properties {
+    availableNetworks: App.Data.Network.NetworkWithCollectionsData[];
+    selectedChainIds: number[];
+    handleSelectedChainIds: (chainId: number) => void;
+}
+
+const NetworkFilterCheckbox = ({
+    handleClick,
+    count,
+    value,
+    isSelected,
+}: {
+    handleClick: (value: number) => void;
+    count: number;
+    value: App.Data.Network.NetworkWithCollectionsData["chainId"];
+    isSelected: boolean;
+}): JSX.Element => (
+    <label className="group flex cursor-pointer items-center justify-between text-sm">
```

Comment:
would be good if the cursor changes to `not-allowed` when it's disabled, currently it still shows cursor-pointer instance while you can't interact with it